transforms/csdl/preprocess_csdl.xsl

```diff
@@ -939,6 +951,44 @@
         </xsl:element>
     </xsl:template>
 
+     <!-- Remove navigability for singleValueExtendedProperties and multiValueExtendedProperties-->
+    <xsl:template match="edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='calendar']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='event']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='contactFolder']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='contact']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='mailFolder']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='message']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='post']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='note']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='outlookTask']/edm:NavigationProperty[@Name='singleValueExtendedProperties']|
+                        edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='outlookTaskFolder']/edm:NavigationProperty[@Name='singleValueExtendedProperties']" >
```

Comment:
We can simplify the match to target all entityTypes:
```suggestion
    <xsl:template match="edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name]/edm:NavigationProperty[@Name='singleValueExtendedProperties']" >
```

Reply:
Thank you for this 👊🏽